packages/teams-js/src/private/files.ts

```diff
@@ -529,4 +593,206 @@ export namespace files {
 
     sendMessageToParent('files.openDownloadFolder', [fileObjectId], callback);
   }
+
+  /**
+   * @hidden
+   * Hide from docs
+   *
+   * Initiates add 3P cloud storage provider flow
+   *
+   * @param callback Callback that will be triggered post add 3P cloud storage provider action
+   */
+  export function add3PCloudStorageProvider(callback: (error?: SdkError) => void): void {
+    ensureInitialized(FrameContexts.content);
+
+    if (!callback) {
+      throw new Error('[files.add3PCloudStorageProvider] callback cannot be null');
```

Comment:
I realize this is a pattern elsewhere in the file, but it would be really nice if functions used SdkError for everything instead of Error *and* SdkError

Reply:
Following your suggestion changed the Errors to SdkErrors. Hope this deviation in consistency isn't a concern for anybody.